test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com/user';
+const token = process.env.ACCESS_TOKEN;
+
+const urlBase2 = 'https://api.github.com/repos/Maosorio92/miweb/issues';
+const obj = axios.create({
+  url: urlBase2,
```

Comment:
can we create the axios client without the URL param?, in that case we can reuse that client for both URLs, the user URL and the issues url.